test/test_pipeline/components/test_setup_image_augmenter.py

```diff
@@ -10,12 +10,10 @@
 class TestImageAugmenter(unittest.TestCase):
     def test_every_augmenter(self):
         image_augmenter = ImageAugmenter()
-        configuration = image_augmenter.get_hyperparameter_search_space().sample_configuration()
+        configuration = image_augmenter.get_hyperparameter_search_space().get_default_configuration()
```

Comment:
Does the default have every augmentation? I guess then this is fine, but perhaps add a comment here that this is implicitly assumed for this test